kustomization/go.mod

```diff
@@ -0,0 +1,67 @@
+module github.com/fluxcd/pkg/kustomization
+
+go 1.17
+
+require (
+	github.com/drone/envsubst v1.0.3
+	github.com/fluxcd/kustomize-controller/api v0.18.2
```

Comment:
We can't import kustomize-controller in `fluxcd/pkg` as it would result in circular dependencies.

Reply:
The import is in `kustomization_varsub_test.go`, which is isolated by being in its own package `kustomization_test` and won't be imported on the controller side.

Reply:
Please use Unstructured objects and remove this import.